### ISSUE TITLE
js/mock_runtime.js can be removed

### ISSUE THREAD

**Issue by kitsonk** [ry/deno#431]
I belive `js/mock_runtime.js` and the `mock_rutime` build step in `BUILD.gn` can be removed since there is a real runtime to be snapshotted now.

**Comment by kitsonk** [comment]
No, sorry, it appears I am wrong, still used in some testing builds around snapshotting.

**Comment by ry** [comment]
(mock_runtime will stay perminenatly as a way of testing libdeno without involving TypeScript or rust.)
